gateway-service-definitions/src/main/resources/services/livy/0.4.0/service.xml

```diff
@@ -17,9 +17,9 @@
 -->
 <service role="LIVYSERVER" name="livy" version="0.4.0">
```

Comment:
Most likely going to want a new version of the service definitions instead of just modifying what is there. I don't think 0.4.0 of Livy had HA redirects.